plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -16,7 +16,7 @@
  */
 function auto_sizes_prime_attachment_caches( $content ): string {
 	if ( ! is_string( $content ) ) {
-		return '';
+		return ''; // @codeCoverageIgnore
```

Comment:
Ditto. I don't think this should be ignored because technically this is a code path that can be tested.